src/System.Management.Automation/utils/ClrFacade.cs

```diff
@@ -530,17 +530,26 @@ internal static object[] GetCustomAttributes<T>(Assembly assembly)
         #region Encoding
 
         /// <summary>
-        /// Facade for Encoding.Default
+        /// Facade for getting Encoding.Default
         /// </summary>
         internal static Encoding GetDefaultEncoding()
         {
             if (s_defaultEncoding == null)
             {
-#if CORECLR     // Encoding.Default is not in CoreCLR
-                // As suggested by CoreCLR team (tarekms), use latin1 (ISO-8859-1, CodePage 28591) as the default encoding.
-                // We will revisit this if it causes any failures when running tests on Core PS.
-                s_defaultEncoding = Encoding.GetEncoding(28591);
-#else
+#if CORECLR
+
+    #if UNUX    // PowerShell Core on Unix
+                s_defaultEncoding = Encoding.GetEncoding(65001);
+    #else       // PowerShell Core on Windows
+                uint aCp = NativeMethods.GetACP();
```

Comment:
Maybe rename the variable to `currentAnsiCp`? It's helpful to know what `GetACP` does.